ci/create-aro-cluster.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
So whenever I mint a new bash script, I try to use at least `set -euo pipefail` from this little gem of an article: http://redsymbol.net/articles/unofficial-bash-strict-mode/

It makes the script much more predictable, and because it's new, you're not going to break a behavior that the existing logic relied upon.

If you think there's enough value in that article's approach, would you consider applying that `set` line and retesting to ensure that the script behaves as expected?

Reply:
Yep I like that.